apps/src/code-studio/components/ShareAllowedDialog.jsx

```diff
@@ -22,6 +22,7 @@ import QRCode from 'qrcode.react';
 import copyToClipboard from '@cdo/apps/util/copyToClipboard';
 import FontAwesome from '@cdo/apps/templates/FontAwesome';
 import Button from '../../templates/Button';
+import defaultThumbnail from '@cdo/static/projects/project_default.png';
```

Comment:
Curious why you updated image in this file which is not a .storybook file. Did you mean to update `Dialog.story.jsx`?

Reply:
No, this was intentional -- this style works in any context (ie, story or not), it just plays particularly nicely with Storybook, which relies on this style to resolve the paths of assets like images.

The other .`story.jsx` files were the ones where I could find files `/blockly/media` URLs directly in them (ie, by searching for that string in files with the story path suffix). I'm sure there are other places where we could update and remove broken images from Storybook, but I didn't want to go through each component individually to find broken images (a lot of components look pretty dated / maybe don't belong in Storybook, so didn't feel 100% useful to do so).